src/main/scala/io/github/sentenza/hacktoberfest18/MenuIO.scala

```diff
@@ -76,5 +76,4 @@ object MenuIO {
           renderInteractiveMenu()
       }
     }
-  }
```

Comment:
Why did you remove this bracket here?

Reply:
I don't understand what this bracket and the next one close. To me the third bracket closes the MenuIO object and my IDE gives me an error when I include it I just tried checking out your branch to make sure the mistake wasn't coming from my code but it seems to be the same.